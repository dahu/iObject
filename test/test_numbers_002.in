" test iObject Numbers

12345
12345 12345
12345ax
ax12345
ax12345ax
ax 12345 ax

