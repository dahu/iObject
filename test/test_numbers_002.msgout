
"test_numbers_002.in" 
"test_numbers_002.in" 9L, 81C
Warning: terminal cannot highlight
"test_numbers_002.out" 
"test_numbers_002.out" [New] 9L, 46C written
