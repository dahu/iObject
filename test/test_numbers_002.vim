source ../plugin/iObject.vim
edit test_numbers_002.in

" delete unsigned numbers
1
call search('\d', 'W')
normal dan
call search('\d', 'W')
normal dan
call search('\d', 'W')
normal dan
call search('\d', 'W')
normal dan
call search('\d', 'W')
normal dan
call search('\d', 'W')
normal dan
call search('\d', 'W')
normal dan

write test_numbers_002.out
quit!
