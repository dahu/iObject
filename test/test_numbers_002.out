" test iObject Numbers


 
ax
ax
axax
ax  ax

